### DISCUSSION TITLE
[Feature] Exclude already stacked photos from duplicate detection

### DISCUSSION THREAD

**Discussion by cuu508** [immich-app/immich#13850] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

In my Immich library I have JPG+RAW pairs already stacked. The duplicate detection feature detects some of them as duplicates. Example:

![image](https://github.com/user-attachments/assets/902c9a14-5bb8-494e-a85a-a26d2c3d724b)

Both photos are already in a stack (I believe in the same one). It would be nice if the duplicate detection feature ignored stacks like that either by default or as an option.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by CyberCreator** [comment]
It would be cool to combine such photos into one stack. That is, you go to the photo card, and there you can scroll through all its versions in different formats.

**Comment by Aliang-code** [comment]
So is this feature in the roadmap now?

**Reply by bo0tzz** [reply]
You tell me, do you see it on https://immich.app/roadmap ? :stuck_out_tongue:

**Comment by bo0tzz** [comment]
#16380
